docusaurus/docs-glossary/glossary.md

```diff
@@ -4,12 +4,19 @@ id: glossary
 # Glossary
 Here you will find a list of terms you may come across as you are using Ziti.
 
+## Edge Router Policy
+An Edge Router Policy is a policy/configuration that describes which Identities can connect to which Edge Routers

```

Comment:
I prefer to say that identities "use" edge routers because "connect" implies intent when in fact, the intention is to connect to the service, and the identity is using the router to do so. It seems best to sidestep the fact that the SDK is connecting to the router via underlay because we're trying to describe the overlay function of the ERP.

Reply:
I dunno, personally I think identities connect to edge routers before connecting to services and this policy is about what routers an identity is permitted to connect to -- to "use". So for me, I feel like it's relevant in this context to say connect.